src/service/poc_mobile.proto

```diff
@@ -7,6 +7,81 @@ import "mapper.proto";
 import "hex_boosting.proto";
 import "service_provider.proto";
 
+message hex_usage_counts_req_v1 {
+  // H3 resolution 10 tile
+  uint64 hex = 1;
+  // Number of unique service provider subscribers active in the hex
+  // during the epoch ( inclusive of discovery mapping enabled subscribers )
+  uint64 service_provider_subscriber_count = 2;
+  // Number of unique discovery mapping enabled subscribers active
+  // in the hex during the epoch
+  uint64 disco_mapping_count = 3;
+  // Number of unique offload users active in the hex during the epoch
+  uint64 offload_count = 4;
```

Comment:
if i understand correctly, these 3 fields represent quantities of users in various categories. Consider a common naming convention between the fields to indicate this... ie: `xxxx_user_count` or `xxx_subscriber_count`

Reply:
addressed here: https://github.com/helium/proto/pull/428/commits/1b9d9548a6f7c4d390cf0585b2d7c61f564e0d8a